model/src/main/kotlin/config/CopyrightGarbage.kt

```diff
@@ -32,12 +32,21 @@ data class CopyrightGarbage(
      * A set of literal strings that identify garbage Copyright findings.
      */
     @JsonSerialize(converter = StringSortedSetConverter::class)
-    val items: Set<String> = emptySet()
+    val items: Set<String> = emptySet(),
+
+    /**
+     * A set of [Regex] patterns that identify garbage Copyright findings.
+     */
+    @JsonSerialize(converter = StringSortedSetConverter::class)
+    val patterns: Set<String> = emptySet()
 ) {
+    private val regex by lazy { patterns.joinToString("|", "(", ")").toRegex() }
```

Comment:
I'm not 100% sure if the string joining always yields the correct regex. It'd be great if this was reviewed by another person.